index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@100&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <div id="wrap">
+    <header class="header">
+        <nav class="nav-bar">
+            <a id="logo" href="#"><img  src="nav-Logo.png" alt="logo"></a>
+            <nav id="nav-icon"><button class="button"><img class="icon" src="nav-Icon.png" alt="menu"></button></nav>
+        </nav>
+     </header>
```

Comment:
- [ ] Good job @vikas924,  I like your desing. But for the logo Please put it outside of the `nav` tag but still inside the `header` tag. This is for semantic reasons and the `nav` bar should wrap only your navigations (menus),